public/javascripts/SVLabel/src/SVLabel/task/TaskContainer.js

```diff
@@ -493,6 +493,17 @@ function TaskContainer (navigationModel, neighborhoodModel, streetViewService, s
         }
     };
 
+    /**
+     * Get street id of current task
```

Comment:
Make sure to double check the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide) for comment style! There should just be a period at the end of the sentence ;)